### PR TITLE
chore: stop code rabbit spam

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #XXX

## What is the purpose of the change

Partial improvement to the code rabbit spam. It still posts on every single PR, but at least its only once.

### CODE REVIEW

File: .coderabbit.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    enabled: false
+    ignore_title_keywords:
+      - "WIP"
+      - "DO NOT MERGE"
+    drafts: false
+    base_branches:
+      - "main"
+      - "v*.x"
+chat:
+  auto_reply: false
```
Comment (ValarDragon) [inline .coderabbit.yaml:19]: oh was this bad?

File: .coderabbit.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+language: "en"
+early_access: false
+reviews:
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
```
Comment (ValarDragon) [inline .coderabbit.yaml:6]: If your annoyed by the size, can turn this to false

File: .coderabbit.yaml (hunk 1 above)
Comment (czarcas7ic) [reply .coderabbit.yaml:19]: Reading the description it implies that setting this to false will only let code rabbit post if it is spoken to, but it seems like that isn't the case

**Comment by czarcas7ic** [comment]
^ is there any repo level settings that are non yaml that will prevent this from getting posted every time?

**Pull request closed without merging**